src/main/kotlin/org/rust/ide/refactoring/extractFunction/ExtractFunctionParameterTablePanel.kt

```diff
@@ -44,10 +101,14 @@ class ExtractFunctionParameterTablePanel(nameValidator: (String) -> Boolean,
                                          private val config: RsExtractFunctionConfig,
                                          private val onChange: () -> Unit)
     : AbstractParameterTablePanel<ParameterDataHolder>(
+    ChooseColumn(),
     NameColumn(nameValidator),
     TypeColumn(project)
 ) {
     init {
+        myTable.setDefaultRenderer(JBCheckBox::class.java, ChooseRenderer())
```

Comment:
Maybe override `org.rust.ide.refactoring.extractFunction.ChooseColumn#getRenderer` instead of call `setDefaultRenderer`? As well as editor.

Reply:
I tried override the `org.rust.ide.refactoring.extractFunction.ChooseColumn#getRenderer` method at first, but it didn't work.

Am I missing something?

Reply:
You're right, it doesn't work for some reason. Well, I think this code could look like this:
```Kotlin
myTable.setDefaultRenderer(Boolean::class.java, BooleanTableCellRenderer())
myTable.setDefaultEditor(Boolean::class.java, BooleanTableCellEditor())
myTable.columnModel.getColumn(0).preferredWidth = WIDTH
```